lib/network/modules/InteractionHandler.js

```diff
@@ -705,6 +708,28 @@ class InteractionHandler {
       this.body.emitter.emit('hidePopup');
     }
   }
+
+
+  /**
+   * Ensure zoom min/max values are acceptable
+   * If they aren't, remove them from `options`
+   * @private
+   */
+  _checkRemoveZoomMinMax(options) {
+    const { zoomMin, zoomMax } = options;
+
+    const notValid = value => (
+      !util.isNumber(value) || Number.isNaN(value) || value < 0
```

Comment:
Why not test for:
```javascript
!(util.isNumber(value) && value >= 0)
```
It's simpler, more readable and covers everything (test all the valid cases, there's generally way fewer of them than the invalid ones and considerable less risk that you'll forget something).

Also is zero zoom really okay? It wasn't allowed before.